modules/install-consul/install-consul

```diff
@@ -77,17 +77,33 @@ function install_supervisord_debian {
   sudo systemctl enable supervisor
 }
 
+function two_way_symlink() {
+  src=$1
+  dest=$2
+
+  if [[ ! -f "$dest" ]] && [[ ! -f "$src" ]]; then
+    echo "Missing source AND destination... exiting..."
```

Comment:
Please print out `$src` and `$dest` here to help with understanding what the error is referring to.